draft-ietf-tls-tls13.md

```diff
@@ -4037,6 +4046,26 @@ lacking a "server_name" extension by terminating the connection with a
 Security issues are discussed throughout this memo, especially in
 {{implementation-notes}}, {{backward-compatibility}}, and {{security-analysis}}.
 
+## Re-using Ephemeral shares
+
+This document refers to client and server key shares as ephemeral. This literally
+means that they are used for a short time. Keeping the private key secret forever
+is necessary for the property of Perfect Forward Secrecy. The simplest and safest 
+way to achieve this is to generate a new Diffie-Hellman share for each TLS 
+handshake. This has some cost in processing power. 
```

Comment:
I'm changing the text, but you want to destroy the keys after use whether they are single-use or not.